packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.workspace.tsx

```diff
@@ -229,25 +228,16 @@ const ImmunizationsForm: React.FC<DefaultPatientWorkspaceProps> = ({
                   name="vaccinationDate"
                   control={control}
                   render={({ field: { onChange, value } }) => (
-                    <DatePicker
+                    <OpenmrsDatePicker
                       id="vaccinationDate"
                       maxDate={new Date().toISOString()}
-                      dateFormat={datePickerFormat}
```

Comment:
Please remove the `datePickerFormat` definition since its no longer required.

Reply:
@NethmiRodrigo, the changes you are requesting have already been fixed. Could I be missing out on something?

Reply:
Yep, you're right! Sorry I missed that!